common/static/js/vendor/jquery.leanModal.js

```diff
@@ -0,0 +1,61 @@
+// leanModal v1.1 by Ray Stone - http://finelysliced.com.au
```

Comment:
I'm slightly nervous about including the unminified version now. Admittedly this is a tiny library, but it still seems unfortunate. Can we include it in one of the bundles so that it automatically gets minified and then we reduce the number of script includes too?